string frame_id
string personId    #Unique id assigned for each person
#float32 distanceFromPR2  #Distance between the person and PR2 base
float64 t0		   	
float64 t1
float32 x #Position (x)
float32 y #Position (y) 
float32 distance_travelled   #Distance travelled by the person from time t0 to time t1
float32 speed	   #Speed of the person
float32 xdistance  #Distance of the person from pr2 along X axis
float32 ydistance  #Distance of the person from pr2 along Y axis
